### DISCUSSION TITLE
How to access the hyperlink in the cell

### DISCUSSION THREAD

**Discussion by hchaozhe** [nithinmurali/pygsheets#492] [Q&A] [answered]
Thank you for generating this library! Very useful!
The spreadsheet I have contains hyperlink to in the cell (associated with text, but not explicitly shown as text)
I was wondering if there is a way to get this hyperlink? 
If I do get_col I simple get the text in the cell.
Thanks!

**Comment by nithinmurali** [comment, marked as answer]
Hi, you can use `cell.hyperlink` to get the hyperlink.

`wks.cell('A1').hyperlink
`

**Reply by hchaozhe** [reply]
Thank you very much!
